shared/blob_utils.py

```
@@ -1,50 +1,138 @@
 import os
 import json
+import aiohttp
 from azure.storage.blob.aio import BlobServiceClient
 from azure.storage.blob.aio._blob_client_async import BlobClient
```

Pylint (3.3.7)

[error] Unable to import 'azure.storage.blob.aio._blob_client_async'

(E0401)